astacus/coordinator/plugins/clickhouse/plugin.py

```diff
@@ -66,21 +66,21 @@ class ClickHousePlugin(CoordinatorPlugin):
     freeze_name: str = "astacus"
     disks: Sequence[DiskConfiguration] = [DiskConfiguration(type=DiskType.local, path=Path(""), name="default")]
     drop_databases_timeout: float = 300.0
-    max_concurrent_drop_databases: int = 100
+    max_concurrent_drop_databases_per_node: int = 10
     create_databases_timeout: float = 60.0
-    max_concurrent_create_databases: int = 100
+    max_concurrent_create_databases_per_node: int = 10
     sync_databases_timeout: float = 60.0
     restart_replica_timeout: float = 300.0
-    max_concurrent_restart_replica: int = 100
+    max_concurrent_restart_replica_per_node: int = 10
     restore_replica_timeout: float = 300.0
-    max_concurrent_restore_replica: int = 100
+    max_concurrent_restore_replica_per_node: int = 10
     freeze_timeout: float = 3600.0
     unfreeze_timeout: float = 3600.0
     # Deprecated parameter, ignored
     attach_timeout: float = 300.0
-    max_concurrent_attach: int = 100
+    max_concurrent_attach_per_node: int = 10
     sync_tables_timeout: float = 3600.0
-    max_concurrent_sync: int = 100
+    max_concurrent_sync_per_node: int = 10
```

Comment:
All the old parameters should be kept (and ignored) to avoid failing if a newer astacus is accidentally installed in a place where the config still uses the old parameters.

I don't know how/why, but I misplaced the `# Deprecated parameter, ignored` comment, it should be just before `use_system_unfreeze` (all the attach/sync parameters are not deprecated). But you can move it in the right place and then put all the new deprecated options after that.

Reply:
Thanks Kevin! Changed.
(again, sorry for the delay with the changes)

Reply:
Nothing to be sorry about :)